src/wallets/hardware/LedgerWallet.ts

```diff
@@ -351,21 +327,18 @@ export default class LedgerWallet
       }),
     });
 
-    const signedSerializedTx = signedTransaction.encode();
+    const res = await this.provider.sendTransaction(signedTransaction);
 
-    const base64Response: any = await provider.sendJsonRpc(
-      "broadcast_tx_commit",
-      [Buffer.from(signedSerializedTx).toString("base64")]
-    );
+    if (typeof res.status !== "string") {
```

Comment:
So after removing the `any` type here, it seems `status` can be either an object or a string. I need your help here @amirsaran3 to understand what the original code was trying to do as previously a crash was possible when `res.status` is a string for whatever reason (`Buffer.from` expects a string not undefined)